Develop

mfirka opened the pull request:
- [DEMO LINK](https://mfirka.github.io/layout_landing-page/)

volodymyr-soltys97 reviewed (changes requested):
Good job 👍 
Let's improve your page
1. Fix the background color for the logo 
<img width="1209" alt="image" src="https://github.com/user-attachments/assets/9a4019c5-3415-444a-b750-6724da0af9d1">

2. Remove scroll when the burger menu is open 
<img width="1512" alt="image" src="https://github.com/user-attachments/assets/9490192d-00a2-4cf1-942b-f54808e87da4">

3. Fix font styles here, check it on the layout 
<img width="1331" alt="image" src="https://github.com/user-attachments/assets/23440317-61c2-4a32-8fdf-edc556473872">

4. Change the autofill styles for the form fields
https://css-tricks.com/snippets/css/change-autocomplete-styles-webkit-browsers/ 
<img width="584" alt="image" src="https://github.com/user-attachments/assets/10a7cd8f-6427-4cf5-9001-2f6c8eeae6c8">

5. The user cannot be able to change size of these fields 
<img width="1334" alt="image" src="https://github.com/user-attachments/assets/8c704bce-5294-408a-8274-af26bad10d41">

6. Hide the phone number on the mobile and tablet versions 
<img width="591" alt="image" src="https://github.com/user-attachments/assets/ba4404aa-74c0-45e6-ad87-7fb1ecbb5566">

mfirka commented:
- Fixed the background color for the logo
- Removed the scroll when the hamburger menu is open
- Fixed font styles to match the layout
- Changed autofill styles for form fields
- Removed the ability to resize the field.
- The phone number is hidden on mobile and tablet versions

volodymyr-soltys97 reviewed (changes requested):
Almost done!
To improve:
1. You need to make a call when user clicking on this icon 
<img width="80" alt="image" src="https://github.com/user-attachments/assets/40bbc318-e26a-4ea7-b4ad-5027913e465e">

2. Add scroll behavior smooth to the HTML selector

3. When you now move the cursor over the inputs, all the fields jump a little, this needs to be fixed

4. The text must have are same color everywhere and fix the border when input isn't focused
<img width="618" alt="image" src="https://github.com/user-attachments/assets/73f1d4f4-abfe-4448-86ca-d55206be2f53">

mfirka commented:
- Clicking on the icon makes a call
- Added smooth scrolling to the HTML selector
- Fixed field jumps when hovering over input
- Text has the same color, fixed frame when input is not highlighted

TarasHoliuk commented:
Almost perfect 👍 

Now fields jump when making one of them focused. Hint: instead of adding border on focus add it by default and just change its color on focus 🙂

TarasHoliuk reviewed (changes requested):
Great job! One tiny improvement required

mfirka commented:
- Fixed frame jumping when focusing

anastasiiavorobiova reviewed (approved):
Awesome!
